packages/react-composites/src/composites/CallWithChatComposite/ChatButtonWithUnreadMessagesBadge.tsx

```diff
@@ -27,8 +26,8 @@ export interface ChatButtonWithUnreadMessagesBadgeProps extends ControlBarButton
 const validNewChatMessage = (message): boolean =>
   !!message.senderDisplayName && (message.type === 'text' || message.type === 'html');
 
-const filledIcon = <Chat20Filled key={'chatIconKey'} primaryFill="currentColor" />;
-const regularIcon = <Chat20Regular key={'chatIconKey'} primaryFill="currentColor" />;
+const filledIcon = <Icon iconName={'ControlBarChatButtonFilled'} />;
```

Comment:
nit: `Icon` I believe is deprecated, use `FontIcon` here and throughout PR
![image](https://user-images.githubusercontent.com/2684369/155364027-581782ec-6be3-433b-970c-7ef65d94e22a.png)

Reply:
ah okay will change that!